pyproject.toml

```diff
@@ -13,7 +13,18 @@ force-exclude = true
 ignore = [
     "D203", # no-blank-line-before-class
     "D212", # multi-line-summary-first-line
+    "D407", # removed dashes lines under sections
     "D417", # argument description in docstring (unreliable)
+    'COM812', # ruff-format recommended
+    'COM819', # ruff-format recommended
+    'D206', # ruff-format recommended
+    'E501', # ruff-format recommended
+    'ISC001', # ruff-format recommended
+    'Q000', # ruff-format recommended
+    'Q001', # ruff-format recommended
+    'Q002', # ruff-format recommended
+    'Q003', # ruff-format recommended
+    'W191', # ruff-format recommended
```

Comment:
Are all of these needed for the ticket #27 as written? Or are you snowballing?

Reply:
#206 was introduced. But when I ran it on these files, it gave a warning. Probably wasn't an issue in `main` as no python files.